HelloWorld/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<manifest package="net.boondockradio.helloworld"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name" android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+
+        <activity android:name=".MainActivity">
```

Comment:
So many times Ive done the same 😆 
ALWAYS declare `Activity`.
You dont need to declare `Fragments`